### PR TITLE
update(docs): sync the docs with chaos-chart via embedmd command

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by:  shubham.chaudhary@mayadata.io

- sync the sample engine & RBAC manifest of docs and chaos-chart via `embedmd` command

### CODE REVIEW

File: website/versioned_docs/version-1.1.0/pod-network-latency.md [hunk 1]
```diff
@@ -187,22 +187,22 @@ spec:
...
+
+            #Network interface inside target container
             - name: NETWORK_INTERFACE
-              value: 'eth0'                   
+              value: 'eth0'     
+
             - name: LIB_IMAGE
               value: 'gaiaadm/pumba:0.6.5'
+
             - name: NETWORK_LATENCY
```
Comment (uditgaurav) [inline website/versioned_docs/version-1.1.0/pod-network-latency.md]: unit

File: docs/openebs-target-network-delay.md [hunk 2]
```diff
@@ -233,14 +233,18 @@ spec:
...
               value: 'cstor-istgt'
+
             - name: APP_PVC
               value: 'demo-nginx-claim'    
+
             - name: DEPLOY_TYPE
-              value: 'deployment'       
+              value: 'deployment'   
+
             - name: NETWORK_DELAY
```
Comment (uditgaurav) [inline docs/openebs-target-network-delay.md]: unit

**Comment by uditgaurav** [comment]
Whenever you will have the next PR in chos-charts try to add the unit of `NETWORK_DELAY` unit.

**Review by uditgaurav — APPROVED** [review]
LGTM

**Review by ksatchit — APPROVED** [review]

**Pull request merged**